Source/Core/Core/CMakeLists.txt

```diff
@@ -272,6 +272,12 @@ set(LIBS ${LIBS} ${MBEDTLS_LIBRARIES})
 
 if(WIN32)
 	set(SRCS ${SRCS} HW/BBA-TAP/TAP_Win32.cpp HW/WiimoteReal/IOWin.cpp)
+	list(APPEND LIBS
+		videod3d
```

Comment:
Well, this is the WIN32 section.
And the file is already a mess anyway. We should have the backends dependencies on some "video" interface target anyway.